packages/insomnia-app/app/ui/components/templating/tag-editor.js

```diff
@@ -103,6 +103,23 @@ class TagEditor extends React.PureComponent<Props, State> {
     );
   }
 
+  _sortRequests(requests: Array<models.request.type | models.requestGroup.type>, parentId: string) {
+    let sortedReqs = [];
+    requests
+      .filter(request => request.parentId === parentId)
+      .sort((a, b) => {
+        if (a.metaSortKey === b.metaSortKey) return a._id > b._id ? -1 : 1;
+        else return a.metaSortKey < b.metaSortKey ? -1 : 1;
+      })
```

Comment:
#2738 is almost complete and it creates a `metaSortKeySort` helper, linked below. Ideally that is used here, so once that PR is merged in we can update this.

https://github.com/cobwebsonsale/insomnia/blob/bca044f74d8bf437b6775ac5a9cf4c9f5f52ba3d/packages/insomnia-app/app/common/sorting.js#L92-L98

Then, this code would just become:
```suggestion
      .sort(metaSortKeySort)
```

Reply:
Sorry for the delay, l lost sight of this PR hehe. Anyways I added the required changes :D